### DISCUSSION TITLE
Gravity Forms integration

### DISCUSSION THREAD

**Discussion by andresclua** [swup/swup#938] [Help] [answered]
### Describe your problem 🧐

Hi! I'm having an issue with Gravity Forms on my website. When I visit the URL https://rubysrainbow.org/donate/ directly, everything works fine. However, if I go to the homepage and then click the "Donate" button, I encounter a problem that forces me to use data-no-swup to refresh the page.

I suspect this issue is related to Gravity Forms or its Stripe integration. Any advice or insights on how to resolve this would be greatly appreciated.

![captura_de_pantalla_2024-07-30_a_las_15 20 20_480](https://github.com/user-attachments/assets/56419a1f-fcd6-48bb-9836-96e13f5f1174)
![captura_de_pantalla_2024-07-30_a_las_15 19 29](https://github.com/user-attachments/assets/ca02a96a-fa36-4c25-8aa1-e0bef8c0f376)

### What is your question? 🙋‍♀️

I would like to have a regular page transition as it is across the site.

### URL to minimal reproduction 🔗

https://rubysrainbow.org/

### Checked all these? 📚

- [X] I have read and searched the [official docs](https://swup.js.org)
- [X] I have checked [discussions](https://github.com/orgs/swup/discussions) and [existing issues](https://github.com/issues?q=is:issue+org:swup) for answers to my question
- [X] I agree to follow this project's [Code of Conduct](https://github.com/swup/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have provided all necessary information to the best of my knowledge

**Comment by daun** [comment, marked as answer]
There was a [recent discussion about Gravity Forms](https://github.com/orgs/swup/discussions/919) and its lack of support for usage with client-side routing. Somebody posted a solution which might be of help.
